apps/internal-ui/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,40 @@
+import clsx from 'clsx';
+
+import { Icon } from '@/components/Icon';
+import { COLOR_STYLES_MAPPER, COLOR_VARIANTS } from '@/variants';
+import { ChipProps } from './types';
+
+const Chip = ({
+  label,
+  onClick,
+  disabled,
+  className,
+  color = COLOR_VARIANTS.PRIMARY_06,
+  backgroundColor = COLOR_VARIANTS.PRIMARY_02,
+  ariaLabel,
+}: ChipProps) => {
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
```suggestion
  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```
q: 혹시 React.MouseEvent 등으로 import 없이 사용하는 부분의 경우도 컨벤션으로 정리하면 좋지 않을까 해서 의견 드립니다.
```
1. React.MouseEvent
2. import { MouseEvent } from 'react'
```
어느 방식이 더 나으신가요?

Reply:
아 이거 자동완성으로 진행되면서 신경 못썼네요 무조건 2번입니다!
이 부분 ESLint에도 강제할 수 있는 린트가 없어서 `no-restricted-syntax`로 대체해 보도록 하곘습니다.

Reply:
이 부분 no-restricted-syntax로 진행해 보려했는데 안잡히고 다른 lint를 더 찾아봤는데 명확히 안나와서 시간이 너무 소요되고있어서 컨벤션 업데이트만 진행하고 추후 찾아보도록 하겠습니다.

Reply:
ref. https://thebbodek.atlassian.net/wiki/spaces/SD/pages/352256035/React+JSX#%ED%83%80%EC%9E%85-%EC%B0%B8%EC%A1%B0